lib/datev/export.rb

```diff
@@ -18,7 +18,7 @@ def <<(attributes)
     def to_s
       CSV.generate(CSV_OPTIONS) do |csv|
         write(csv)
-      end
+      end.encode(Encoding::WINDOWS_1252)
```

Comment:
Why do we need this? Specs are running fine on Linux (at Travis) and my local OS X machine without this change. Encoding is already set to `windows-1252` by the `CSV_OPTIONS` hash.